packages/lesswrong/components/tagging/CoreTagsSection.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Absolute beauty of a component. It is simple, but you did the Right Thing with the grid and it's all very :chefs-kiss: